front/src/components/list-of-group/index.js

```diff
@@ -0,0 +1,3 @@
+import GroupListPage from './GroupListPage';
```

Comment:
Why do we need `GroupListPage.js` at all? Please, move all the code from `GroupListPage.js` to the index.js inside list-of-group` folder

Reply:
done